demo/simple.html

```diff
@@ -502,10 +499,7 @@
 					</d2l-tbody>
 				</d2l-table>
 			</div>
-		</div>`;
-
-document.body.appendChild($_documentContainer.content);
-</script>
+		</div>
```

Comment:
The bug that polymer-modulizer is trying to fix here does not apply in this case. The tests pass.